language-support/ts/codegen/tests/daml/Main.daml

```diff
@@ -52,3 +53,5 @@ data Foo a b = Foo {
   y: Foo b a;
   z: b
 }
+
+data Color = Red | Blue | Yellow
```

Comment:
Can we please have a test that this works. I suggest adding a field `color: Color` to the `AllTypes` template.
I would also prefer to add a field of type `[Color]` to see that the `Color` object generated by the code really contains a `decode` function and can hence be passed to the `List` function imported from `daml-json-types`.

Reply:
that's a good idea, I'll implement it on monday.